src/test/java/org/jitsi/videobridge/cc/EndpointMultiRankerTest.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright @ 2015 - Present, 8x8 Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jitsi.videobridge.cc;
+
+import org.jitsi.videobridge.*;
+import org.junit.*;
+
+import java.util.*;
+import org.jitsi.videobridge.cc.BitrateController.*;
+
+import static org.junit.Assert.*;
+
+public class EndpointMultiRankerTest
+{
+    final List<EndpointMultiRank> endpointMultiRanks = new ArrayList<>();
+    final VideoConstraints stageVideoConstraints
+        = new VideoConstraints(/* idealHeight */ 720);
+    final VideoConstraints thumbnailVideoConstraints
+        = new VideoConstraints(/* idealHeight */ 180);
+
+    @Test
+    public void followActiveSpeaker()
+    {
+        EndpointMultiRank activeSpeakerMultiRank
+            = makeNextSpeakerWithVideoConstraints(stageVideoConstraints);
```

Comment:
nit: I think the naming of these variables (`activeSpeakerMultiRank`, `speaker2MultiRank`, `speaker3MultiRank`) are maybe a bit confusing, because (as the next test shows) this code prioritizes based on their ideal height, not whether or not they're the active speaker (which this code isn't even aware of).

Reply:
active speaker rank is the tie breaker if both the ideal and preferred height is equal and the order in which the endpoints are added to the list determines their speaker rank. I should probably add this note in the code

Reply:
@bbaldino I've improved the comment that explains the assertions. Let me know if it's better now

Reply:
🤦 I didn't realize speaker rank was part of that structure, sorry!  Thanks.